src/App.js

```diff
@@ -1,26 +1,120 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import Main from './components/Main';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: '',
+      items: [],
+      selectedList: 0,
+      checked: false,
+    };
+
+    this.addToValue = (event) => {
+      this.setState({
+        value: event.target.value,
+      });
+    };
+
+    this.addToItems = () => {
+      if (this.state.value) {
```

Comment:
Suggestion: You can use opposite condition and not wrap whole code in the `if`
```suggestion
      if (!this.state.value) {
        return;
      }
```